### PR TITLE
Added UIB Manager

### PR DESCRIPTION
**Pull request by redgreenbird**

### CODE REVIEW

File: Runtime/Utils/UIBuildManager.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using UnityEngine.UIElements;
+
+namespace Mixin.Utils
+{
+    public class UIBuildManager<T> : Singleton<T> where T : Singleton<T>
```
Comment (Nixtrin) [inline Runtime/Utils/UIBuildManager.cs]: "where T : Singleton<T>"
Is this neccesary?

**Review by Nixtrin — COMMENTED** [review]
"where T : Singleton"
Is this neccesary?

File: Runtime/Utils/UIBuildManager.cs (hunk 1 above)
Comment (redgreenbird) [reply Runtime/Utils/UIBuildManager.cs]: i tried without but it didnt work. maybe you can have a try?

Comment (Nixtrin) [reply Runtime/Utils/UIBuildManager.cs]: `public class UIBuildManager<T> : Singleton<UIBuildManager<T>>`

Comment (Nixtrin) [reply Runtime/Utils/UIBuildManager.cs]: `public class UIBuildManager : Singleton<UIBuildManager>`

**Pull request merged**